druid/examples/readme.md

```diff
@@ -55,12 +55,31 @@ cargo run --example hello
 This shows some of the basics of druid. If you need a start of how to build an application with a text-box and some labels this is where to start.
 
 ## Hello_web
+Not working, browser is blank after going to http://localhost:8000?
+
+For more info and prerequistes see [druid/examples/hello_web/README.md](druid/examples/hello_web/README.md).
 ```
 cd druid/examples/hello_web
 wasm-pack build --out-dir pkg --out-name hello_web
+http
 ```
+[View at http://localhost:8000](http://localhost:8000].
+
 This is an example of how to get almost any druid application can be used on the web. This is just the hello_world example but should work for all of them.
 
+## Web
+Not working, browser is blank after going to http://localhost:8000?
```

Comment:
Does this work for other people, as with the /web example it does not seem to work for me?

Reply:
As Hello_web above, it works for me now.